src/turnkeyml/analyze/script.py

```diff
@@ -182,13 +177,18 @@ def explore_invocation(
         fs.Keys.PARAMETERS,
         model_info.params,
     )
+    stats.save_stat(
+        fs.Keys.CLASS,
+        type(model_info.model).__name__,
+    )
     if fs.Keys.AUTHOR in tracer_args.labels:
         stats.save_stat(fs.Keys.AUTHOR, tracer_args.labels[fs.Keys.AUTHOR][0])
-    if fs.Keys.CLASS in tracer_args.labels:
-        stats.save_stat(fs.Keys.CLASS, tracer_args.labels[fs.Keys.CLASS][0])
     if fs.Keys.TASK in tracer_args.labels:
         stats.save_stat(fs.Keys.TASK, tracer_args.labels[fs.Keys.TASK][0])
```

Comment:
If these are going to be removed from the top level stats, the omz could should change to index in to the labels dict to get this info.
And omz will contain yaml files from the old version and the new version which will complicate the web front end.
Is it possible to add each label as a top-level stats? So we dont remove any existing labels but rather add to it.

Reply:
> And omz will contain yaml files from the old version and the new version which will complicate the web front end.

Ok you're right, I shouldn't make breaking changes to reserved fields the stats file.

> Is it possible to add each label as a top-level stats?

I'm nervous about collisions if we automatically add all labels, because the labels API is arbitrary. For example, someone could add a `model_name` label that would collide with, and override, our `model_name` reserved field. 

Let's add specific labels to the top-level on a case by case basis. For example, we should add `license`. Let's do that in a separate PR though - please file an issue if that is a feature you want for OMZ.